CHANGELOG.md

```diff
@@ -1,4 +1,80 @@
+- chore: added database\_name as deprecated attribute [\#129](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/129) ([PacoDw](https://github.com/PacoDw))
+- Encryption At Rest: fixed issues and added an enhancement [\#128](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/128) ([PacoDw](https://github.com/PacoDw))
+- Add PIT enabled argumento to Cluster Resource and Data Source [\#126](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/126) ([marinsalinas](https://github.com/marinsalinas))
+- X509 Authentication Database User [\#125](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/125) ([PacoDw](https://github.com/PacoDw))
+- Database users: added x509\_type attribute [\#122](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/122) ([PacoDw](https://github.com/PacoDw))
+- Shared tier doc edits [\#119](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/119) ([themantissa](https://github.com/themantissa))
+- Private endpoints [\#118](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/118) ([PacoDw](https://github.com/PacoDw))
+- Update cluster doc [\#117](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/117) ([themantissa](https://github.com/themantissa))
+- Update backup, add links [\#116](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/116) ([themantissa](https://github.com/themantissa))
+- Projects: adding teams attribute [\#113](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/113) ([PacoDw](https://github.com/PacoDw))
+- Update cluster.html.markdown [\#112](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/112) ([themantissa](https://github.com/themantissa))
+- Fix DiskSizeGB missing [\#111](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/111) ([marinsalinas](https://github.com/marinsalinas))
+- Terraform resource for MongoDB Custom Roles [\#108](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/108) ([PacoDw](https://github.com/PacoDw))
+- Fix peering resource [\#107](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/107) ([PacoDw](https://github.com/PacoDw))
+- Fix \#68: Added the ability to re-create the whitelist entry when it's remove manually [\#106](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/106) ([PacoDw](https://github.com/PacoDw))
+- Updating `git clone` command to reference current repository [\#103](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/103) ([macintacos](https://github.com/macintacos))
+- Cluster label and plugin attribute [\#102](https://github.com/terraform-providers/terraform-provider-mongodbatlas/pull/102) ([PacoDw](https://github.com/PacoDw))
```

Comment:
Move to
Implemented enhancements to existing resources:

Reply:
both lines?

Reply:
Only 56 but 55 can move there as well.